### DISCUSSION TITLE
Duda Datasets proyecto

### DISCUSSION THREAD

**Discussion by joseehernandez** [rramosp/ai4eng.v1#156] [UDEA Students]
Buenas noches Profesor , espero esté muy bien.
Queria preguntarle cual de los datasets de las siguientes competiciones nos recomienda usar para el proyecto, y si estrictamente el dataset encontrado tiene que cumplir con los requisitos?
https://www.kaggle.com/competitions/how-much-did-it-rain-ii/data
https://www.kaggle.com/competitions/tabular-playground-series-apr-2022/data
https://www.kaggle.com/competitions/store-sales-time-series-forecasting/data?select=holidays_events.csv4

Muchas gracias.

**Comment by rramosp** [comment]
te recomendaría el primero y el tercero. verifica si cumple las condiciones
y, si no es el caso, puedes proponer alguna alternativa como se explica en
la página del curso (simulando datos faltantes, etc.)

On Fri, Jul 1, 2022 at 10:40 PM joseehernandez ***@***.***>
wrote:

> Buenas noches Profesor , espero esté muy bien.
> Queria preguntarle cual de los datasets de las siguientes competiciones
> nos recomienda usar para el proyecto, y si estrictamente el dataset
> encontrado tiene que cumplir con los requisitos?
> https://www.kaggle.com/competitions/how-much-did-it-rain-ii/data
> https://www.kaggle.com/competitions/tabular-playground-series-apr-2022/data
>
> https://www.kaggle.com/competitions/store-sales-time-series-forecasting/data?select=holidays_events.csv4
>
> Muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/156>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKXW2GTPDE2OAYFLGTVR6225ANCNFSM52OMMMLA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
